src/contexts/alert_handler.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { createContext, useContext, useState } from "react";
+
+const AlertHandlerContext = createContext();
+
+export const AlertHandlerProvider = ({ children }) => {
+  const [alertState, setAlertState] = useState({
+    isActive: false,
+    alertMessage: "",
+  });
+
+  const setAlert = (alertMessage) => {
+    setAlertState({ isActive: true, alertMessage: alertMessage });
+  };
+
+  const closeAlert = () => {
+    setAlertState({ isActive: false, errorText: "" });
```

Comment:
```suggestion
    setAlertState({ ...alertState, isActive: false });
```

もしくは

```suggestion
    setAlertState({ isActive: false, errorText: alertState.errorText });
```

こうすると以前のerrorテキストは現在のままで、isActiveだけ更新することができると思います。

Reply:
@higashiji2000 
提案ありがとうございます!
修正しました!